build-tools/xaprepare/xaprepare/Application/Utilities.cs

```diff
@@ -446,6 +446,12 @@ public static void FormatSize (ulong dSize, out decimal value, out string unit)
 			value = SignificantDigits (value, 3);
 		}
 
+		public static string SizeToString (long dSize)
```

Comment:
I used `long` here because the `DriveInfo` type uses `long` for its properties. I would rather cast to `ulong` in this method rather than at the call site.